documentation/smg/18_herbert_merger_proposal.md

```diff
@@ -0,0 +1,70 @@
+# Procedure to amalgamate the Herbert repository into Horace
+
+### Feb 1. 2022
+
+## Introduction
+
+The development team recently agreed that it would be a good idea to amalgamate the
+[Herbert](https://github.com/pace-neutrons/Herbert/)
+and [Horace](https://github.com/pace-neutrons/Horace) repositories,
+in order to avoid duplication of settings, start-up routines and CI tests.
+
+This system management guide proposes a procedure to accomplish this amalgamation.
+The requirements of this procedure is that:
+
+* It should not impact other developments of Horace / Herbert
+* It should retain the histories of both repositories
+* Continuous integration builds and tests should work seamlessly across the transition.
+
+In addition to these requirements there is a major secondary goal of reducing the
+size of the repository (currently ~1.3GB for *Horace* and ~110MB for *Herbert*),
+which in particular means removing or reducing the ~850MB of test files in *Horace*,
+to alternative storage as noted in
+[ADD09](https://github.com/pace-neutrons/Horace/blob/7c9e35b08831212ff23656832c81162261a20226/documentation/add/09_test_data_storage.md),
+and then amending the repo history in order excise these files and thus reduce the repo size.
+A minor secondary goal is to rename the default branch to `main` to keep in
+consistency with new naming conventions.
```

Comment:
I didn't realize we'd done this. What is the reference for the new conventions?

Reply:
I guess it's because `master` has bad connotations, but a lot of projects have been renaming the default branch to `main` - it doesn't really matter to `git` since it's meant to be branch agnostic. *Github* requires a default branch though. What we call it is up to us... For the proposed amalgamation steps I need to create new branches so that development in the old `master` can continue in parallel... I just thought it would also be a good time to rename the default branch.

Reply:
I was hating political correctness talks from the USSR time. To me the statement "The XXI Counsel of Communistic Party of the Soviet Union is requesting us to develop modern radiation resistant material" which I had to include in my PhD to show that I am a law abiding person to avoid troubles with our political leadership, and the current statement that "The poor blacks are offended by the mention `master`" and requesting us to rename it to avoid troubles with ??? are related to  the same degree of insanity. For this reason I would keep the `master` as it is. 

Here we are all slaves working for our `master` and I like this connotation.